openapi/paths/storefront/kyc-liveness-sessions@{id}@finish.yaml

```
@@ -8,12 +8,17 @@ post:
   summary: Finish KYC liveness session
   operationId: StorefrontPostKycLivenessSessionFinish
   x-sdk-operation-name: finish
-  description: Attempt to finalize a pending KYC liveness session.
+  description: |-
+    Attempt to finalize a pending KYC liveness session.
```

nitpick: [Start all operation descriptions with an active verb](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)
```suggestion
    Attempts to finalize a pending KYC liveness session.
```